pages/_app.tsx

```diff
@@ -2,39 +2,28 @@ import '../styles/globals.css';
 
 import type { AppProps } from 'next/app';
 import Head from 'next/head';
+import { appWithTranslation } from 'next-i18next';
 import React from 'react';
 
 import AlgoliaSearch from '@/components/AlgoliaSearch';
 import ScrollButton from '@/components/buttons/ScrollButton';
 import Banner from '@/components/campaigns/Banner';
 import Footer from '@/components/footer/Footer';
 import Layout from '@/components/layout/Layout';
-import { MDXProvider } from '@/components/MDX/MDX';
 import NavBar from '@/components/navigation/NavBar';
 import StickyNavbar from '@/components/navigation/StickyNavbar';
 import AppContext from '@/context/AppContext';
-import { defaultLanguage, defaultNamespace, I18nProvider, languages, namespaces } from '@/utils/i18n';
-import loadLocales from '@/utils/locales';
 
 /**
  * @description The MyApp component is the root component for the application.
  */
 function MyApp({ Component, pageProps, router }: AppProps) {
-  const i18n = {
-    languages,
-    defaultLanguage,
-    namespaces,
-    defaultNamespace,
-    locales: loadLocales()
-  };
-
   return (
-    <I18nProvider i18n={i18n}>
       <AppContext.Provider value={{ path: router.asPath }}>
+        {/* <MDXProvider components={mdxComponents}> */}
```

Comment:
Why MDXProvider is commented?

Reply:
We are now using the [MDX provider for the next.js](https://nextjs.org/docs/pages/building-your-application/configuring/mdx) instead of the provider from the `@mdx-js/react` library. 
you can find the configuration of the new next.js MDX Provider [here](https://github.com/asyncapi/website/pull/3111/files#diff-00760c96c632d58f2c4862219a590c7fb40fa5d8c2365c3d26c3cf8e8d95012e). 

When we kept provider from `@mdx-js/react`. We ran into several MDX errors during the build. (see screenshot)

![image](https://github.com/user-attachments/assets/fbfcd547-7a2f-4d97-b141-39466e30070e)

@sambhavgupta0705 I think we should delete the commented lines

Reply:
So, are we now changing the whole MDX rendering part as well?

Reply:
Yes correct. 
Please note that, it does not affect end result. Rendered MDX will look exactly as it did with the previous MDXProvider.